### PR TITLE
Fix aftman regex pattern and update some tags to resolve deprecation warnings.

### LINKED ISSUE #1: This if for foreman and not for aftman.
Like the title says, either changing the repo name or updating the renovate.json would resolve a lot of confusion for others.

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2021 sasial-dev
+Copyright (c) 2024 kineticwallet
```
Comment (sasial-dev) [inline LICENSE]: Can you revert this?

File: README.md [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
-A renovate config for [Aftman](https://github.com/LPGhatguy/aftman).
+A renovate config for [Aftman](https://github.com/LPGhatguy/aftman) & [Foreman](https://github.com/Roblox/foreman).
 
 It can be added by changing your config file:
 
 ```json
 {
     "extends": [
-      "github>sasial-dev/aftman-renovate-config"
+      "github>kineticwallet/aftman-renovate-config"
```
Comment (sasial-dev) [inline README.md]: and this!

File: default.json [hunk 3]
```diff
@@ -2,10 +2,11 @@
     "extends": [
       "config:base"
     ],
-    "regexManagers": [
+    "customManagers": [
       {
+        "customType": "regex",
         "fileMatch": ["^aftman.toml$"],
-        "matchStrings": [".+ = { name = \"(?<depName>.+)\", version = \"(?<currentValue>.+)\" }", ".+ = (?<depName>.+)@(?<currentValue>.+)"],
+        "matchStrings": [".+ = { name = \"(?<depName>.+)\", version = \"(?<currentValue>.+)\" }", ".+ = \"(?<depName>.+)@(?<currentValue>.+)\""],
```
Comment (sasial-dev) [inline default.json:9]: Thank you for this!

**Review by sasial-dev — CHANGES_REQUESTED** [review]
Minor things - and thank you for this!

**Comment by ghost** [comment]
Wouldn't it be possible to only merge .json files?

**Comment by ghost** [comment]
LGTM to merge now. @sasial-dev

**Review by sasial-dev — APPROVED** [review]
Thank you!

**Pull request merged**